src/Microsoft.DotNet.Build.Tasks.Feed/BlobUrlInfo.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Auth;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    class BlobUrlInfo
+    {
+        private const string accountNameandEndpointRegex = @"(?<accountname>[a-z0-9]+)\.(?<endpoint>.+?)";
+        private const string containerAndBlobRegex = @"/(?<containername>[^\/]+)/(?<blobpath>.*)";
+        public string AccountName { get; set; }
+
+        public string Endpoint { get; set; }
+
+        public string ContainerName { get; set; }
+
+        public string BlobPath { get; set; }
+
+        public bool HasToken { get; set; }
+
+        public Uri Uri
+        {
+            get
+            {
+                return new Uri($"https://{AccountName}.{Endpoint}/{ContainerName}/{BlobPath}");
```

Comment:
`public Uri Uri => return new...`?

Reply:
Changed this instead to store the passed in  URI, or construct on the fly via the other constructor